docs/content/SplitPageLayout.mdx

```diff
@@ -0,0 +1,313 @@
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### With pane on right
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Pane position="end">
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### Without padding
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header padding="none">
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane padding="none">
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content padding="none">
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer padding="none">
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### Without header or footer
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Pane>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+</SplitPageLayout>
+```
+
+### With non-sticky pane
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane sticky={false}>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+## Props
+
+### SplitPageLayout
+
+<PropsTable>
+  <PropsTableSxRow />
+</PropsTable>
+
+### SplitPageLayout.Header
+
+<PropsTable>
+  <PropsTableRow
+    name="divider"
+    // TODO: Document ResponsiveValue
+    // This prop accepts a viewport range map (i.e. {regular: ..., narrow: ...}) in addition to a single value.
+    // The `regular` and `wide` keys of the viewport range map can be 'none' or 'line'.
+    // The `narrow` key of the viewport range map can be 'none' or 'line' or 'filled'.
+    type={`ResponsiveValue<
+  'none' | 'line',
+  'none' | 'line' | 'filled'
+>`}
+    defaultValue="'line'"
+  />
+  <PropsTableRow
+    name="visible"
+    type={`ResponsiveValue<boolean>`}
+    defaultValue="true"
+    description="Whether the header is visible."
+  />
+  <PropsTableRow
+    name="padding"
+    type={`| 'none'
+| 'normal'
+| 'condensed'`}
+    defaultValue="'normal'"
+  />
+  <PropsTableSxRow />
+</PropsTable>
+
+### SplitPageLayout.Content
+
+<PropsTable>
+  <PropsTableRow
+    name="width"
```

Comment:
Should this be called "maxWidth" if it's the **maximum** width of the content area?

Reply:
That makes sense to me. We'd need to update this in PageLayout as well. @vdepizzol @jonrohan How you feel about `width` -> `maxWidth`?

Reply:
I think it should remain as `width`. As a prop, it's a characteristic of the region, not a reference to the CSS attribute. Responsive components are expected to adjust on smaller viewport sizes anyway.